client/ayon_core/hosts/maya/plugins/publish/extract_pointcache.py

```diff
@@ -152,24 +201,226 @@ def process(self, instance):
+
+        # The Arguments that can be modified by the Publisher
+        overrides = set(cls.overrides)
+
+        # What we have set in the Settings as defaults.
+        flags = set(cls.flags)
+
+        enabled_flags = [x for x in flags if x in overrides]
+        flags = overrides - set(override_defs.keys())
+
+        tooltips = {
+            "autoSubd": (
+                "If this flag is present and the mesh has crease edges, crease"
+                " vertices or holes, the mesh (OPolyMesh) would now be written"
+                " out as an OSubD and crease info will be stored in the "
+                "Alembic  file. Otherwise, creases info won't be preserved in "
+                "Alembic file unless a custom Boolean attribute "
+                "SubDivisionMesh has been added to mesh node and its value is "
+                "true."
+            ),
+            "dontSkipUnwrittenFrames": (
+                "When evaluating multiple translate jobs, this decides whether"
+                " to evaluate frames between jobs when there is a gap in their"
+                " frame ranges."
+            ),
+            "eulerFilter": "Apply Euler filter while sampling rotations.",
+            "noNormals": (
+                "Present normal data for Alembic poly meshes will not be "
+                "written."
+            ),
+            "preRoll": "This frame range will not be sampled.",
+            "renderableOnly": "Only export renderable visible shapes.",
+            "stripNamespaces": (
+                "Namespaces will be stripped off of the node before being "
+                "written to Alembic."
+            ),
+            "uvWrite": (
+                "Uv data for PolyMesh and SubD shapes will be written to the "
+                "Alembic file."
+            ),
+            "uvsOnly": (
+                "If this flag is present, only uv data for PolyMesh and SubD "
+                "shapes will be written to the Alembic file."
+            ),
+            "verbose": "Prints the current frame that is being evaluated.",
+            "wholeFrameGeo": (
+                "Data for geometry will only be written out on whole frames."
+            ),
+            "worldSpace": "Any root nodes will be stored in world space.",
+            "writeColorSets": "Write vertex colors with the geometry.",
+            "writeFaceSets": "Write face sets with the geometry.",
+            "writeNormals": "Write normals with the deforming geometry.",
+            "writeUVSets": (
+                "Write all uv sets on MFnMeshes as vector 2 indexed geometry"
+                " parameters with face varying scope."
+            ),
+            "writeVisibility": (
+                "Visibility state will be stored in the Alembic file. "
+                "Otherwise everything written out is treated as visible."
+            )
+        }
+        tooltip = ""
+        for flag in flags:
+            tooltip += "{} - {}\n".format(flag, tooltips[flag])
```

Comment:
This introduces a trailing new line currently.

![image](https://github.com/ynput/ayon-core/assets/2439881/feeb872b-a5d2-447b-b29a-275743eb4520)


```suggestion
        tooltip = "\n".join(f"{flag} - {tooltips['flag']}" for flag in flags)
```

Also, because this is based on sets you're losing the sorted nature of the flags in the tooltip:
![image](https://github.com/ynput/ayon-core/assets/2439881/1dafc613-d96b-482f-9c2c-41ce37ef1ab4)

As such, you're better off making sure the flags are sorted for readability.
e.g.
```suggestion
        tooltip = "\n".join(f"{flag} - {tooltips['flag']}" for flag in sorted(flags))
```